sdk/core/core-amqp/src/util/utils.ts

```diff
@@ -64,6 +64,7 @@ export interface ServiceBusConnectionStringModel {
   Endpoint: string;
   SharedAccessKeyName: string;
   SharedAccessKey: string;
+  SharedAccessSignature?: string;
   EntityPath?: string;
   [x: string]: any;
```

Comment:
We should probably be strict here and say that our interface doesn't include "the other bits" (ie, the parser is not obligated to deliver anything more than the attributes we know about).